frappe/contacts/doctype/address/address.py

```diff
@@ -39,6 +39,7 @@ def autoname(self):
 	def validate(self):
 		self.link_address()
 		self.validate_reference()
+		self.set_link_title()
```

Comment:
Maybe put this in the `address_and_contact.py` file? Since it's being used in both places?

Reply:
Set this individually in the controller so that the field is populated even if the contact or address is programatically created through get_doc or the API